### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by rmorlang**
This PR will make things more awesome because:
* even cooler now!

Before merging this PR, I will make sure of the following:
- [ ] all the eyes are dotted
- [x] all the tees are crossed
- [x] every me has been read
- [x] wheels have been reinvented and squared

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 do not read this
+Fri 22 Sep 2023 12:40:23 AM UTC
```
Comment (omazhary) [inline README.md:2]: - [ ] More todos for you to do.

**Comment by rmorlang** [comment]
@whole-weet help

**Comment by rmorlang** [comment]
@whole-weet ignore

**Comment by rmorlang** [comment]
@whole-weet I am sorry

**Pull request closed without merging**